trpcultivate_phenotypes/src/Plugin/TripalImporter/TripalCultivatePhenotypesTraitsImporter.php

```diff
@@ -102,92 +137,80 @@ public static function create(ContainerInterface $container, array $configuratio
    *
    * @param array $form_values
    *   An array of the importer form values provided to formValidate.
+   * @param string $file_mime_type
+   *   A string of the MIME type of the input file, usually grabbed from the
+   *   file object using $file->getMimeType()
+   *
    * @return array
    *   A listing of configured validator objects first keyed by
    *   their inputType. More specifically,
    *   - [inputType]: and array of validator instances. Not an
    *     associative array although the keys do indicate what
    *     order they should be run in.
    */
-  public function configureValidators($form_values) {
+  public function configureValidators(array $form_values, string $file_mime_type) {
 
     $validators = [];
 
     // Setup the plugin manager
     $manager = \Drupal::service('plugin.manager.trpcultivate_validator');
 
-    // Importer assets.
-    // All values will be accessible to every instance of the validator Plugin.
-    // This importer does not require a project and this variable is set to 0
-    // instruct validators Project + Genus that relations project-genus can be ignored.
-    $project = 0;
+    // Grab the genus from our form to use in configuring some validators
     $genus = $form_values['genus'];
-    $file_id = $form_values['file_upload'];
-
-    // Make the header columns into a simplified array where the header names
-    // are the values
-    $headers = array_keys($this->headers);
-
-    // Take our simplified headers array and flip the array keys and values
-    // This is the format that the validators will expect to know which indices
-    // in the row of data to act on
-    // For example: ['Trait Name'] => 0
-    $header_index = array_flip($headers);
 
```

Comment:
// Load the file object.
```suggestion
    // Open the file
```

Reply:
After discussing in person, this suggestion was meant for a previous commit and has since been resolved